datalad/core/local/status.py

```diff
@@ -473,3 +473,6 @@ def custom_result_summary_renderer(results):  # pragma: more cover
                         len(annexed),
                         single_or_plural('file', 'files', len(annexed)),
                         total_size))
+        if all(r.get('state', None) == 'clean' for r in results):
```

Comment:
Is it valid to assume that every result coming through is `action='status'`?  It looks like code just upstream does not make this assumption.

Reply:
Good Q. Will investigate and add a filter, if needed. Thx!

Reply:
I have added the check. Even if it is not necessary now, we run the risk of silently breaking it in the future, if it is no ensured.